app/assets/javascripts/Components/Modals/send_lti_grades_modal.jsx

```diff
@@ -0,0 +1,96 @@
+import React from "react";
+import Modal from "react-modal";
+
+class LtiGradeModal extends React.Component {
+  static defaultProps = {
+    override: false,
+  };
+
+  constructor(props) {
+    super(props);
+    this.state = {
+      deployments: this.props.lti_deployments,
+    };
+    this.handleChange = this.handleChange.bind(this);
```

Comment:
I'm almost certain this isn't necessary. What problem are you solving by adding this?